test/validation/api/ipsec/ipsec.c

```diff
@@ -815,26 +947,46 @@ static int ipsec_send_out_one(const ipsec_test_part *part,
 			ev = recv_event(queue, 0);
 			if (ODP_EVENT_INVALID != ev) {
 				CU_ASSERT(odp_event_is_valid(ev) == 1);
+
 				CU_ASSERT_EQUAL(ODP_EVENT_PACKET,
 						odp_event_types(ev, &subtype));
-				CU_ASSERT_EQUAL(ODP_EVENT_PACKET_BASIC,
-						subtype);
-				CU_ASSERT(!part->out[i].status.error.all);
+
+				if (ODP_EVENT_PACKET_IPSEC != subtype) {
+					CU_ASSERT_EQUAL(ODP_EVENT_PACKET_BASIC,
+							subtype);
+					CU_ASSERT(!part->out[i].status.error.all);
+				}
 
 				pkto[i] = odp_packet_from_event(ev);
 				CU_ASSERT_FATAL(pkto[i] != ODP_PACKET_INVALID);
+
+				if (part->out[i].sa_expiry != IPSEC_TEST_EXPIRY_IGNORED)
+					ipsec_status_event_get(sa, part->out[i].sa_expiry);
```

Comment:
If status events come when they are ignored, they stay in the queue until read at some later point. This can be problematic. I think this is related to the question on what IGNORE really means.

Reply:
Ack. Using NONE to disable the SA expiry related logic in common code and IGNORE now just ignores the values present in the status result.